tests/backend/user/test_apiv1_human_find.py

```diff
@@ -293,9 +305,9 @@ async def test_pagination(access_testbed, organization_factory, local_device_fac
     assert rep == {
         "status": "ok",
         "results": [
-            {"user_id": blacky.user_id, "human_handle": blacky.human_handle, "revoked": False},
             {"user_id": godfrey1.user_id, "human_handle": godfrey1.human_handle, "revoked": False},
-            {"user_id": mike.user_id, "human_handle": mike.human_handle, "revoked": False},
+            {"user_id": richard.user_id, "human_handle": richard.human_handle, "revoked": False},
```

Comment:
no need for `NonDeterministicOrderedResults` here ?

Reply:
I don't think so because the order should be the same as the insert order.